lib/poll.js

```diff
@@ -4,19 +4,33 @@ export default class Poll extends EE.EventEmitter {
   constructor (options, frequency) {
     super();
     this.url = 'https://' + options.host + '/Remote/GetActiveTableID';
-    this.opts = {
-      shortTableId: options.account + '/' + options.table
-    };
+    this.shortTableId = options.account + '/' + options.table
+
+    if (options.cacheBust) {
+      this.cacheBust = options.cacheBust;
+    } else {
+      this.cacheBust = fasle;
+    }
     this.table = void 0;
     frequency = frequency || 30;
     this.interval = frequency * 1000;
     this.timeout = void 0;
     this.aborted = false;
     this.checkTable();
   }
+  getCacheBust() {
+    // return Math.random().toString().slice(2);
```

Comment:
```suggestion
    // unique integer is generated every second
```

Reply:
the comment is how i originally did it and wondered if we wanted to make that available as an option